src/styles/blocks/stars.scss

```
@@ -0,0 +1,33 @@
+$cardWidth: 200px;
+$secondaryColor: #616070;
+$buttonColor: #00acdc;
+$buttonTextColor: #fff;
+$mainColor : #060b35;
+/* #region stars */
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: 96px;
+}
+
+.stars__star {
```

please use the power of nesting
```suggestion

  &__star {
  //...
}
```